src/diffTool/diffProcessor.ts

```diff
@@ -49,17 +50,9 @@ export async function diffRaml(
  * @returns flattened JSON-LD AMF graph
  */
 async function generateGraph(ramlFilePath: string): Promise<object> {
-  ramlToolLogger.debug(`Generating flattened JSON-LD for ${ramlFilePath}`);
-  amf.plugins.document.WebApi.register();
-  amf.plugins.document.Vocabularies.register();
-  amf.plugins.features.AMFValidation.register();
-  await amf.Core.init();
+  const parsedRaml = await parseRamlFile(ramlFilePath);
 
-  //TODO: use common parser function that is used by validator and sdk
-  let model = await new amf.Raml10Parser().parseFileAsync(
-    `file://${ramlFilePath}`
-  );
-  model = new amf.Raml10Resolver().resolve(model);
+  const model = new amf.Raml10Resolver().resolve(parsedRaml);
 
   const renderOptions = new amf.render.RenderOptions().withoutSourceMaps
```

Comment:
Testing what? Already have test coverage on this method don't we?

Reply:
how abt a test that
expects parseRamlFile is called with ramlFilePath..

Reply:
We actually talked about that...the problem we both see with that is that you are kind of testing the backend implementation in a way when you do that.  Which leads to a flaky test.  Basically, if we change that it would break the test but really no break anything except the test.  You want tests that when the tests break the signify some broken functionality in the code. I'm not sure doing that would do. 

 (Mind you hey said he might still write a test that does this on a branch to test out how to use sinon and such.)

Reply:
I am looking this like, 

- use X instead of Y (For some reason)
- Ensure X is used as expected

Reply:
But doesn't that create a test failure when there is no functionality failure?  Which really just becomes a roadblock to changing implementations out?

Reply:
Spoke with @kalagarraj offline, he’s happy to resolve this conversation.